scripts/generate-reference-files-md.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/no-dynamic-require */
+
+/**
+ * Generate markdown for reference Sketch files. The contents of the
+ * reference-files/{sketchVersion} folders and the plugin's manifest.json file
+ * are taken as the source of truth.
+ */
+
+const { resolve } = require('path')
+const { readdirSync } = require('fs')
+const table = require('markdown-table')
+const { writeFileSync } = require('fs')
+const { execSync } = require('child_process')
+
+const dir = resolve(__dirname, '../reference-files')
+const pages = resolve(__dirname, '../docs/pages')
+const pluginPath = resolve(dir, './plugin.sketchplugin')
+const manifest = require(resolve(pluginPath, 'Contents/Sketch/manifest.json'))
+const repoPath =
```

Comment:
I think you can take `https://github.com/BohemianCoding/SketchAPI` from the package.json

Reply:
Yep.

Reply:
Hmm, the `respository.url` field in package.json is for urls to the actual `.git` repo, not the http location of the GitHub web ui. Going to add the `homepage` field to package.json, and grab from there ...